docs/content/en/docs/migrate/strategy/_index.md

```diff
@@ -12,5 +12,259 @@ but the format and content is still evolving.
+  resources need to be located
+  in the same namespace as the associated
+  [KeptnMetric](../../yaml-crd-ref/metric.md)
+  resources.
+  But
+  [KeptnEvaluationDefinition](../../yaml-crd-ref/evaluationdefinition.md)
+  resources that are used for pre- and post-deployment
+  can reference metrics from any namespace.
+  So you can create `KeptnMetrics` in a centralized namespace
+  (such as `keptn-lifecycle-toolkit`)
+  and access those metrics in evaluations on all namespaces in the cluster.
+* Each `KeptnApp` resource identifies the namespace to which it belongs.
+  If you configure multiple namespaces,
+  you can have `KeptnApp` resources with the same name
+  in multiple namespaces without having them conflict.
+* You do not need separate namespaces for separate versions of your application.
+  The `KeptnApp` resource includes fields to define
+  the `version` as well as a `revision`
+  (used if you have to rerun a deployment
+  but want to retain the version number).
+
+So, possible namespace designs run the gamut:
+
+* Move all the Keptn v1 projects you had into a single namespace
+* Move a group of Keptn v1 projects into a single namespace
+* Define one namespace per Keptn v1 project
+* Define one namespace per Keptn v1 stage
+
+## Disposition of Keptn v1 components in KLT
+
+To help you wrap your mind around the process,
+this lists Keptn v1 components
+and identifies their possible disposition when you migrate to KLT.
+
+### project
+
+A Keptn v1 project represents an arbitrary, higher-level grouping of services.
+A project is defined by a
+[shipyard.yaml](https://keptn.sh/docs/1.0.x/reference/files/shipyard/)
+file.
+
+KLT does not recognize projects.
+Some Keptn v1 projects may translate into `KeptnApp` resources
+but many will not.
+A project has a 1:1 mapping to a Git repo,
+but, since few applications are stored in a mono-repo,
+it is difficult to implement a project-as-application paradigm
+
+In general, the Keptn v1 project is a useful as a reference list
+when migrating to KLT
+but does not directly translate to a KLT resource.
+
+### service
+
+A Keptn v1 service models a smaller chunk of a project.
+Most projects include many services
+In a micro-services world,
+a service may represent a micro-service
+but it could instead be a wrapper for something else,
+like "the entire public website"
+
+KLT does not have the concept of a service.
+When migrating to KLT,
+you need to analyze what each service is doing
+and translate that into an appropriate resource.
+The closest analogy is a Kubernetes
+[orkload](https://kubernetes.io/docs/concepts/workloads/)
+but some services may become
+[Keptn tasks](../../implementing/tasks/)
+or other resources.
+
+### stage
+
+A stage is a subsection of a project
+and has no corresponding KLT component.
+However, the logic of the stages can be useful
+when architecting the migration:
+
+* **deployment stage** -- KLT does not itself deploy the software
```

Comment:
this sentence may be equivocal, in a sense KLT does deploy your app since it stops k8s and allows to bind pods to nodes...   I would maybe phrase it stating that: KLT does handle only the single application lifetime, it is not a continuous delivery tool since it does not provide promotion, thus for KLT there is no concept of stage. What it does is prevent the scheduling/deployment of an application version based on user-defined tasks/evaluations and integrate with other continuous delivery tools.

Reply:
I incorporated some of this info here and some at the beginning of the page where we discuss the main paradigm shifts for KLT versus Keptnv1.